### PR TITLE
Add MacOS and Linux set up guide

### PR DESCRIPTION
**Pull request by LionZXY**
# What's new

- Add set up guide for MacOS
- Add set up guide for Linux
- Remove -j flag from instruction (not yet needed)

# Verification 

- Find clear Mac without any library
- Follow guide
- Check that all command execute correctly

- Find clear Debian/Ubuntu without any library
- Follow guide
- Check that all command execute correctly

# Checklist (do not modify)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -129,6 +130,41 @@ That's exactly how we generate our `full` builds.
...
+```
+git clone git@github.com:rusdacent/hex2dfu.git
+cd hex2dfu
+gcc hex2dfu.c ED25519/*.c -o hex2dfu
+sudo cp hex2dfu /usr/local/bin
+```
+
+### Compile firmware
+```
+make -j$(nproc)
```
Comment (skotopes) [inline ReadMe.md]: `nproc` is not available on mac os

**Review by skotopes — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 1 above)
Comment (LionZXY) [reply ReadMe.md:155]: O_o
image

**Comment by LionZXY** [comment]
TODO:
- Add dfu-util instruction
- Add linux instruction
- Remove -j flag from instruction

**Review by skotopes — APPROVED** [review]

**Comment by skotopes** [comment]
Going to be fixed in #827

**Pull request closed without merging**